rita_common/src/dashboard/wallet.rs

```diff
@@ -40,7 +40,7 @@ fn withdraw_handler(address: Address, amount: Option<Uint256>) -> HttpResponse {
     let mut amount = if let Some(amount) = amount {
         amount
     } else {
-        balance.clone()
+        balance.clone().unwrap_or_else(|| 0u64.into())
```

Comment:
if balance is none return an error that we're not yet ready to withdraw. (that should probably be above line 40 becuase we can't send any amount if we don't yet know our balance.